kimhaejung/controllers/likeController.js

```diff
@@ -0,0 +1,14 @@
+const likeService = require("../services/likeService");
+
+const createLikes = async (req, res) => {
+  try {
+    const { userId, postId } = req.body;
```

Comment:
토큰 확인하는 함수에서 마지막에 userId를 정의한 부분이 있을 거에요. 그것을 활요하면 다음과 같이 userId를 불러올 수 있습니다.

```suggestion
    const userId = req.userId
    const { postId } = req.body;
```